api/src/pcapi/routes/backoffice_v3/templates/offerer/get/details/managed_venues.html

```diff
@@ -17,6 +18,7 @@
         <td class="text-break">{{ links.build_venue_name_to_pc_pro_link(venue, public_name=true) }}</td>
         <td>{{ links.build_siret_to_external_link(venue) }}</td>
         <td>{{ venue.venueTypeCode.value }}</td>
+        <td>{{ venue.dms_adage_status | format_dms_status if venue.dms_adage_status else "-"  }}</td>
```

Comment:
```suggestion
        <td>{{ venue.dms_adage_status | format_dms_status if venue.dms_adage_status else ""  }}</td>
```
En général on laisse la case vide quand il n'y a rien à afficher.